seatunnel-dist/pom.xml

```diff
@@ -380,6 +380,11 @@
                     <artifactId>connector-openmldb</artifactId>
                     <version>${project.version}</version>
                 </dependency>
+                <dependency>
+                    <groupId>org.apache.seatunnel</groupId>
+                    <artifactId>connector-doris</artifactId>
+                    <version>${project.version}</version>
```

Comment:
```suggestion
                    <version>${project.version}</version>
                    <scope>provided</scope>
```

Reply:
Thanks for your advice. I have fixed it.